src/etc/inc/services.inc

```diff
@@ -134,7 +134,7 @@ function services_radvd_configure($blacklist = array())
         }
 
         $ifcfgipv6 = get_interface_ipv6($dhcpv6if);
-        if (!is_ipaddrv6($ifcfgipv6)) {
+        if (!is_ipaddrv6($ifcfgipv6) && !isset($config['interfaces'][$dhcpv6if]['dhcpd6track6allowoverride'])) {
```

Comment:
especially with this in mind...

Reply:
Out earning pennies at moment
I'll have a look when I get home.